README.md

```diff
@@ -1,56 +1,48 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] [OPTIONAL] This Pull Request should have a summary describing introduced changes in more detail, kindly edit the description so it meets this requirement